README.md

```diff
@@ -202,10 +202,27 @@ module.exports = {
 
 ## TypeScript
 
+This guide provides multiple TypeScript configuration options. These configs correlate to the major Node.js versions and configure the appropriate `lib`, `module`, `target`, and `moduleResolution` settings. The table below specifies the config name with the Node.js version.
+
+| Node.js Version | TypeScript Config       |
+| --------------- | ----------------------- |
+| v16             | `typescript/node16`     |
```

Comment:
Can we put the full paths in here, as we do in other sections of the README? i.e. `@vercel/style-guide/typescript/node16`